app/ts/PreferencesTab/Preferences/TradeOff/TradeOffReferenceCriterionStatement/TradeOffSlider/TradeOffSlider.tsx

```diff
@@ -61,34 +61,47 @@ export default function TradeOffSlider({
       <Grid container style={{minWidth: '400px', minHeight: '50px'}}>
         <Grid item xs={2} style={{marginTop: '25px', textAlign: 'center'}}>
           <div>
-            <b>{isDecreasingPvf ? 'To' : 'From'}</b>
+            <b>From</b>
+          </div>
+          <div>
+            {getPercentifiedValue(
+              isDecreasingPvf ? upperBound : lowerBound,
+              usePercentage
+            )}
```

Comment:
lot's of ternaries on the same condition. Would prefer an if else with some consts above the return.